dropshot_endpoint/src/lib.rs

```diff
@@ -935,14 +935,6 @@ mod tests {
             const _: fn() = || {
                 struct NeedRequestContext(<Arc<RequestContext<std::i32> > as dropshot::RequestContextArgument>::Context) ;
             };
-            const _: fn() = || {
-                fn need_extractor<T>()
-                where
-                    T: ?Sized + dropshot::Extractor,
```

Comment:
could this change to `RequestExtractor`?

Reply:
... or maybe `SharedExtractor`?

Reply:
I think it could change to `ExclusiveExtractor` (shared extractors can be treated as exclusive, but not the other way around).  (see my other comment -- I do intend to restore a helpful message for this case, one way or another)